src/components/MessageList/__tests__/__snapshots__/VirtualizedMessageList.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`VirtualizedMessageList should render the list without any message 1`] = `
 <div
-  className="str-chat str-chat-channel messaging light"
+  className="str-chat str-chat-channel messaging light "
```

Comment:
does this extra space need to be there?

Reply:
Was a little easier to just add the space to the snapshot, but I changed it now so that the space is only added if it's needed to add another className